react/components/Slider.tsx

```diff
@@ -10,22 +10,27 @@ import PaginationDots from './PaginationDots'
 
 import sliderCSS from './slider.css'
 
-const Slider: FC = ({ children }) => {
+const Slider: FC<SliderLayoutSiteEditorProps & { totalItems: number }> = ({
```

Comment:
why do not create a new interface that extends `SliderLayoutSiteEditorProps` and add the `totalItems` as a attribute?

Reply:
Nice! Would make it cleaner :)